# origin: Debian (horms)
# cset: n/a
# inclusion: not suitable for upstream
# revision date: 2005-03-31
# description: tidy up qla2xxx non-free prune
--- a/drivers/scsi/Makefile	2005-03-31 16:23:04.000000000 +0900
+++ b/drivers/scsi/Makefile	2005-03-31 16:23:29.000000000 +0900
@@ -79,7 +79,6 @@
 obj-$(CONFIG_SCSI_QLOGIC_ISP)	+= qlogicisp.o 
 obj-$(CONFIG_SCSI_QLOGIC_FC)	+= qlogicfc.o 
 obj-$(CONFIG_SCSI_QLOGIC_1280)	+= qla1280.o 
-obj-$(CONFIG_SCSI_QLA2XXX)	+= qla2xxx/
 obj-$(CONFIG_SCSI_PAS16)	+= pas16.o
 obj-$(CONFIG_SCSI_SEAGATE)	+= seagate.o
 obj-$(CONFIG_SCSI_FD_8xx)	+= seagate.o
--- a/drivers/scsi/Kconfig	2005-03-31 16:32:50.000000000 +0900
+++ b/drivers/scsi/Kconfig	2005-03-31 16:32:53.000000000 +0900
@@ -1340,8 +1340,6 @@
 	  To compile this driver as a module, choose M here: the
 	  module will be called qlogicpti.
 
-source "drivers/scsi/qla2xxx/Kconfig"
-
 config SCSI_SEAGATE
 	tristate "Seagate ST-02 and Future Domain TMC-8xx SCSI support"
 	depends on X86 && ISA && SCSI && BROKEN
